README.md

```diff
@@ -87,85 +88,34 @@ To point the API server to `k8s-dqlite` use the following arguments:
 
 ## Highly Available Dqlite
 
-The following steps allows one to setup a highly available dqlite backend.
-
-In the example below, we will setup a 3 node HA dqlite.
+K8s-dqlite supports high availability by using the Raft protocol.
+Below are the steps to set up a highly available dqlite cluster with 3 nodes.
 
-Steps:
+### Steps
 
-1. On the main node, make sure that this node is listening to the default network interface.
-2. On the joining node and after installing k8s-dqlite, you need to backup the dqlite data directory for example (`/var/data`).
-3. Delete the dqlite data directory.
-4. Copy the `cluster.crt` and `cluster.key` from the main node and place it into the joining node's `/var/data`.
-5. Create the `init.yaml` file with the following content
+1. On the main node, ensure that `k8s-dqlite` is listening to the default network interface.
```

Comment:
How does someone verify this?

Reply:
Great question!
It is not very clear but I assume this refers to the listen address. 
So for this example, we could do `sudo netstat -tuln | grep <the ip of the main node>:29001`